apps/jobs/models.py

```diff
@@ -102,6 +102,11 @@ class Submission(TimeStampedModel):
         null=True,
         blank=True,
     )
+    environment_log_file = models.FileField(
+        upload_to=RandomFileName("submission_files/submission_{id}"),
```

Comment:
Can you change the path of the file here? This should be `submission_files/environment_log_file_{id}`